sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -0,0 +1,133 @@
+<!-- eslint-disable vue/valid-v-slot -->
+<template>
+  <v-card class="mb-3">
+    <v-container>
+      <p class="label-bold-normal mb-0 text-body-2">
```

Comment:
I see the place where the content is kept is different from out general UI/UX standards.
Unless there is a real strong reason from business, we must follow our usual UI/UX.


![image](https://github.com/user-attachments/assets/39270818-0ab1-42ea-9deb-9c6ce0473768)

Other UIs following our standards. 
![image](https://github.com/user-attachments/assets/80db8505-bfa8-43ce-8dd7-c72148da8a57)

![image](https://github.com/user-attachments/assets/3c79fbb9-88f9-417f-bc15-467f3feeb851)

![image](https://github.com/user-attachments/assets/6f5aac27-1ab7-4dd8-aa80-99a12704b3e5)

I have asked the question to business to see if that is really needed this way?

Reply:
We got the reply from business and ACs updated. This whole content section can be ignored. 

![image](https://github.com/user-attachments/assets/71284db5-d47f-4597-8cb4-c33926262eef)